README.md

```diff
@@ -2,42 +2,98 @@ PReview Code frontend [![Build Status](https://travis-ci.org/preview-code/fronte
+Please read the complete README.md file before executing any of the steps.
 
-Make sure you have `web-component-tester` installed globally as well as
-the plugin `web-component-tester-istanbul`:
+If you live in the Netherlands, please consider using the following mirror in "Ubuntu Software & Updates":
+<ftp://ftpserv.tudelft.nl/pub/Linux/archive.ubuntu.com>
 
-```
-sudo npm i -g web-component-tester web-component-tester-istanbul
-```
+# Installation guide
+## Prerequisites: nodejs, npm, and git
+1. Install the LTS version (4.x) of [node.js](https://nodejs.org/en/). Please __do not__ install version 6.0 as it is not officially supported by [polymer](https://www.polymer-project.org/1.0/start/first-element/intro) (last checked on 2016.09.22).
+ 
+		curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+		sudo apt-get install nodejs -y
+2. install the _latest_ version of [npm](https://www.npmjs.com/):
 
-# Installation for serviceWorker
-Service-worker requires you to run the website on `https`. Therefore you need
-to install a local self-signed certificate for `localhost:5200`.
+		sudo npm install npm@latest -g
+3. install [git](https://git-scm.com/):
 
-When running `gulp serve` by default the certificate in `certificate/` is
-loaded and attached. However, when opening the website you will get a message
-that the page is insecure. Perform the following steps to resolve this:
+		sudo apt-get install git -y
+__Please note__: It is important to first install `nodejs`, as shown here, and then `npm`. The script for `nodejs` downloads an older version of `npm`, which we subsequently update.
 
-(This guide is inspired by http://stackoverflow.com/a/15076602/2761676)
+## Install tooling
+* Install Chrome:
 
-1. Open the website, you should see an insecurity warning from Chrome
-* Click on advanced
-* Click on continue anyways
-* Click on the red lock in the url bar
-* Click `Details`
-* Click `View certificate`
-* Click `Details`
-* Click `Export...`
+  	Please visit https://www.google.com/chrome/ to install the newest version of Chrome.
+
+## Install the npm (global) packages
+1. install the [java](http://packages.ubuntu.com/en/trusty/default-jre) runtime
+
+		sudo apt-get install default-jre
+1. install [bower](https://bower.io/)
+
+		sudo npm install -g bower
+2. install [gulp](http://gulpjs.com/)
+		
+        sudo npm install -g gulp-cli
+3. install [Polymer CLI](https://www.polymer-project.org/1.0/)
+		
+        sudo npm install -g polymer-cli
+4. install [polyserve](https://github.com/PolymerLabs/polyserve)
+		
+        sudo npm install -g polyserve
+5. install [web-component-tester](https://github.com/Polymer/web-component-tester)
+		
+        sudo npm install -g web-component-tester@4.2.2
+5. install [web-component-tester-istanbul](https://github.com/thedeeno/web-component-tester-istanbul)
+		
+        sudo npm i -g web-component-tester-istanbul@git://github.com/TimvdLippe/web-component-tester-istanbul.git#coverage-log-test-result-fix
```

Comment:
This PR was merged upstream, you can resort to istanbul plugin normally instead of relying on my fork